CleanWeatherTests/Scenes/PeopleList/PersonDisplayableTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PersonDisplayableTests.swift
+//  CleanWeatherTests
+//
+//  Created by Marek Skrzelowski on 24/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import CleanWeather
+
+final class PersonDisplayableTests: QuickSpec {
+
+    override func spec() {
+
+        var person: [Person]!
```

Comment:
you're goal is to check PersonDisplayable model
so you create a file for that - thats ok
and you check in all describe closures all methods you want to test - in your example only custom Init

so you don't need an array of people. just test all parameters only for 1 object, no need arrays